index.js

```diff
@@ -132,15 +134,16 @@ function detectiveEs6Cjs(ast, detectiveOptions) {
  * @param {Object} [options.fileSystem=undefined] - An alternative fs implementation to use for reading the file path.
  * @return {String[]}
  */
-precinct.paperwork = function(filename, options = {}) {
-  options = Object.assign({
-    includeCore: true
-  }, options);
+precinct.paperwork = (filename, options = {}) => {
+  options = {
+    includeCore: true,
```

Comment:
Can you confirm @mrjoelkemp this is correct? Do you really mean to overwrite the property?

```js
options = {
    ...options,
    includeCore: true
  };
```